# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BHMoveActionGoal action_goal
BHMoveActionResult action_result
BHMoveActionFeedback action_feedback
